examples/mongodb/1.8/Dockerfile

```diff
@@ -2,13 +2,14 @@ FROM mongo:5.0
 
 LABEL maintainer="Debezium Community"
 
-COPY init-inventory.sh /usr/local/bin/
-RUN chmod +x /usr/local/bin/init-inventory.sh
-
 # Starting with MongoDB 4.4 the authentication enabled MongoDB requires a key
 # for intra-replica set communication
 RUN openssl rand -base64 756 > /etc/mongodb.keyfile &&\
-    chown mongodb /etc/mongodb.keyfile &&\
+    chown mongodb:mongodb /etc/mongodb.keyfile &&\
     chmod 400 /etc/mongodb.keyfile
 
-CMD ["mongod", "--replSet", "rs0", "--auth", "--keyFile", "/etc/mongodb.keyfile"]
+COPY start-mongodb.sh /usr/local/bin/
+RUN chmod +x /usr/local/bin/start-mongodb.sh
+
+ENTRYPOINT ["docker-entrypoint.sh"]
+CMD ["start-mongodb.sh"]
```

Comment:
Hey @ismailsimsek, are you sure you want to define both `ENTRYPOINT` **and** `CMD`? When I understand it correct, I will start docker-entrypoint.sh with an the other script as argument, like calling `$ docker-entrypoint.sh start-mongodb.sh`. Is that what you supposed?

Reply:
good  point, removed it

Reply:
I think you should look into the parent [Dockerfile](https://github.com/docker-library/mongo/blob/master/5.0/Dockerfile). You want to use `ENTRYPOINT` as this usally reflects that this container is used alone for that one service (see https://www.ctl.io/developers/blog/post/dockerfile-entrypoint-vs-cmd/) and I think you want to clear `CMD`, which is done when you overwrite `ENTRYPOINT` too.

Reply:
makes sense replaced `CMD` with `ENTRYPOINT`